### ISSUE TITLE
Show user error message when they don't have access to the project/fileview

### ISSUE THREAD

**Issue by ychae** [Sage-Bionetworks/data_curator#338]
**Describe the bug**
When I access a DCA where I don't have access to the underlying Synapse project/fileview I get an endless "Generating..." message in the dropdown values without knowing why it won't resolve. 

**To Reproduce**
1. Go to https://sagebio.shinyapps.io/csbc_data_curator_staging/
2. Click on "Select Your Dataset" on the left
3. Dropdowns in Project, Folder, and Template all display "Generating..." rather than values

**Expected behavior**
When I don't have access to the underlying fileview, I should get an error message letting me know that I need to request access to that fileview. 

**Priority** (select one)
- [ ] Minor ⬇️
- [X] Major 📢
- [ ] Critical 🆘

**Screenshots**
<img width="1063" alt="Screen Shot 2022-06-01 at 3 28 31 PM" src="https://user-images.githubusercontent.com/12868382/171513394-be4844f7-5722-4682-a50f-3bef20379322.png">

**Comment by rrchai** [comment]
@ychae Thanks for reporting. I could reproduce the behavior at csbc staging, but it should not be the expected behavior. Can you point me where the codebase is?

**Comment by ychae** [comment]
@rrchai of course! It's located here: https://github.com/mc2-center/data_curator

**Comment by ychae** [comment]
I should note that I see the same behavior in this staging app as well: https://sagebio.shinyapps.io/include-dcc-data-curator-staging/

Codebase for that is here: https://github.com/include-dcc/data_curator

**Comment by rrchai** [comment]
@ychae yep! There is something suspicious when reading the fileview storage and it is a bug. I will look into it later.

**Comment by ychae** [comment]
@rrchai great, thank you!

**Comment by rrchai** [comment]
Related [ticket](https://github.com/Sage-Bionetworks/data_curator/issues/148)
